### ISSUE TITLE
Talk about newton-raphson being like gradient descent with a non-fixed learning rate.

### ISSUE THREAD

**Issue by edkrueger** [edkrueger/gradient-descent-newton-raphson#6]
Example: https://github.com/edkrueger/num-methods-MLE/blob/master/weibull_mle.ipynb
